source/autoSettingsUtils/utils.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2019 NV Access Limited
+
+"""Utility methods for Driver and AutoSettings instances
+"""
+
+
+def paramToPercent(current, min, max):
+	"""Convert a raw parameter value to a percentage given the current, minimum and maximum raw values.
+	@param current: The current value.
+	@type current: int
+	@param min: The minimum value.
+	@type current: int
+	@param max: The maximum value.
+	@type max: int
+	"""
+	return int(round(float(current - min) / (max - min) * 100))

```

Comment:
In python 3, the behavior of round was changed to return an integer if the number of digits is omitted.

```suggestion
	return round((current - min) / (max - min) * 100)
```